doc/nrf/protocols/bt_mesh/concepts.rst

```diff
@@ -316,3 +316,14 @@ Replay protection
 
 To guard against malicious devices replaying previous messages, every device keeps a running sequence number, which is used for outbound messages.
 Each mesh message is sent with a unique pair of sequence number and source address. When receiving a message, the receiving device stores the sequence number and makes sure that it is more recent than the last sequence number it received from the same source address.
+
+.. _mesh_concepts_coexistence:
+
+Bluetooth LE and mesh coexistence
+*********************************
+
+Bluetooth mesh uses the preset local identity value.
+For more information about the advertisement identity, see :ref:`zephyr:bluetooth_mesh_adv_identity`.
```

Comment:
What is the label referring to? I cannot find this.

Reply:
Is it the Advertisement identity section under mesh core?

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/11553#discussion_r1233864549